common-ethereum/build.gradle

```diff
@@ -0,0 +1,63 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        minSdkVersion 22
+        targetSdkVersion 29
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+
+        // 10.0.2.2 - IP address of the host system on Android emulator
+        // Refer to included Docker Compose to launch the nodes locally
+        buildConfigField "String", "ETH_HTTP_URL", "\"http://10.0.2.2:8545\""
```

Comment:
Won't this break when the IP is different?

Reply:
It will, I will replace it with the link to Goerli testnet node, from what I remember liquidity pool app also used Goerli for their stuff.